### DISCUSSION TITLE
ORDER API vs Report API

### DISCUSSION THREAD

**Discussion by carolinaoxobyruffwear** [amzn/selling-partner-api-models#4230] [Q&A] [answered]
Hello community, 

I am working on getting MFN orders using SP-API. I've research and a few forums recommend using the Report API instead of Get Orders API. 
Can you give me your input on which one to use? if Report, which one specific to get all orders? 
The SP-API documentation states: "The reports are intended for order tracking, not to drive a seller's fulfillment process" so would like any input from other developers.
Thanks!

**Comment by mafge** [comment, marked as answer]
Hi @carolinaoxobyruffwear,
thanks for reaching out!
There is no generic advice on whether you should use Orders API or Reports API over the other, this is very much dependent on your workflows and the specific use cases you need to address on your side. If you want to look into available reports which contain all orders, I recommend to look at the [order tracking reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values-order#order-tracking-reports). These are available in XML and as flat files and you can use whichever suits your needs best.
Best,
Marc
Selling Partner Developer Services

**Reply by chrisskhoury** [reply]
+1 on @mafge's comment. It really depends on what you want to do with the order data.

One thing to highlight if your goal is to initiate a fulfillment flow—where you purchase shipments and deliver the order—then I would also recommend leveraging the [ORDER_CHANGE](https://developer-docs.amazon.com/sp-api/docs/tutorial-subscribe-to-order-change-notification) notification. By subscribing to this notification, you can efficiently track changes to orders, such as status updates or cancellations, and trigger the fulfillment process accordingly. This ensures you receive real-time updates whenever there are changes to your orders. You can streamline your order management and fulfillment processes, ensuring timely and accurate deliveries while keeping customers informed throughout the process.
